docs/guides/installing-on-linux/installing-ams-on-linux.md

```diff
@@ -43,19 +74,19 @@ After you've downloaded or copied the file to the AMS instance, open a terminal
 cd path/to/where/ant-media-server....zip
 ```
 
-## Download the installation script
+#### Download the installation script
 
 Download the ```install_ant-media-server.sh``` shell script.
 
     sudo wget https://raw.githubusercontent.com/ant-media/Scripts/master/install_ant-media-server.sh && sudo chmod 755 install_ant-media-server.sh
```

Comment:
Please put them as code by putting in ``` block

Reply:
I reviewed about suggested changes. I fixed a few more similar blocks and minor formatting.
Thanks, I'm merging this PR for now.